README.md

```diff
@@ -215,3 +215,29 @@ result in build failure.
 All files in the repository must contain a license header in the format mentioned in [License Header](license.header).
 
 The static code analysis framework will also validate if the license exists in the specified format.
+
+### Sensitive Parameters Handling
+
+The xpanse project involves the use of some sensitive information, such as AK/SK sensitive fields in
+credential management, sensitive variable information during service deployment, etc.
+
+* For local development, the AES private key file must exist in the project root path. 
+* For environments where the application jar is directly executed,The AES key must exist under 
```

Comment:
This line is still not very clear. The file must exist right next to the jar file?